docs/gatsby-theme-ocular/api-reference/options.md

```diff
@@ -29,7 +29,7 @@ The following options are available:
 | `EXAMPLES`         | `Array` | See below |
 | `STYLESHEETS`      | `Array` | Additional CSS stylesheets |
 | `INDEX_PAGE_URL` | `String`  | Optional URL to a replacement component for the home page. |
-| `DOC_PAGE_URL` | `String`  | Optional URL to a replacement component for doc pages. |
+| `DOC_MARKDOWN_PAGE_URL` | `String`  | Optional URL to a replacement component for doc pages. |
```

Comment:
This just reflects what was working, `DOC_PAGE_URL` support was broken as far as I could tell.